### PR TITLE
chore: Update all dependencies and use new ESLint config.

### PR DESCRIPTION
**Pull request by leerob**
- Updates to `eslint-config-get-off-my-lawn` v5. 
- Fixes stylelint path and addresses two issues.
- Updates snapshots based on those style changes.
- General dependency upgrade.
- Adds Node version information.

### CODE REVIEW

**Comment by manovotny** [comment]
@leerob yo, check out [eslint-config-get-off-my-lawn@5.0.1](https://github.com/manovotny/eslint-config-get-off-my-lawn/releases/tag/v5.0.1)! 😎

File: .eslintrc.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
-    extends: 'get-off-my-lawn-prettier',
+    extends: 'get-off-my-lawn',
     rules: {
-        'node/no-extraneous-import': '',
-        'import/no-extraneous-dependencies': 'off',
-        'import/no-unresolved': 'off'
+        'node/no-extraneous-import': 0,
+        'import/no-extraneous-dependencies': 0,
+        'unicorn/filename-case': 0
```
[the remark this answers is not in the document]
Comment (leerob) [reply .eslintrc.js]: Addressed.

**Comment by leerob** [comment]
Ready for review again @manovotny 🎉

**Comment by leerob** [comment]
@manovotny Stylelint was pointing to the wrong path. I had to update a couple of lines of CSS to appease the robots 🤖 

https://github.com/Hy-Vee/lerna-yarn-workspaces-monorepo/pull/6/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=dotfile#diff-a0c5927996b17c51e955929c7bc2ec66L16

**Comment by manovotny** [comment]
@leerob when you're right, you're right. My 👀 totally read both files as snapsnots. Didn't see that one was a package file. 😊

**Pull request merged**